sdk/core/azure-core/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Added 'OPTIONS' HTTP method to `Azure::Core::Http::HttpMethod` enum.
```

Comment:
Can you extract this change to Core, and make it a separate PR?
This way, it will make it to the GA release this week, and next week you'll be able to release Tables.

Otherwise, if it only makes it to Core Beta (let's say we do an additional release next week), then we will not be able to publish Tables in vcpkg public repo - only in our Beta package registry, and your package will have less public exposure until its next release.